### ISSUE TITLE
Request precinct-level results for 2014, 2015 and 2016 general election

### ISSUE THREAD

**Issue by ghing** [openelections/openelections-sources-ky#1]
@dwillis  says:

> 
> Kentucky is wide open: we have a single 2014 general file from Bell County. Here's contact info for county clerks: http://elect.ky.gov/contactcountyclerks/pages/default.aspx
> 
> First step is to see if they've posted precinct-level results (the state has county-level). if so, grab them and make a pull request here:
> 
> https://github.com/openelections/openelections-sources-ky
> 
> If not, send them a request for precinct-level results for the 2014, 2015 and 2016 primary and general elections (KY did governor and other offices in 2015). If they ask for money, fwd that to openelections@gmail.com.
> 
> Thanks!

**Comment by ghing** [comment]
It appears that precinct-level results for the 2016 general election are available in a Clarity system at http://results.enr.clarityelections.com/KY/64180/182267/Web01/en/summary.html.  However, the site labels these as "unofficial" results.  I could see a few courses of action here:

* Check with a few clerks and see if the Clarity system represents their most official results, even if they're not officially official. 
* Start parsing 2016 precinct results from Clarity data because unofficial results are good enough and better than nothing 
* Focus on other elections and ignore 2016 for now because, hey, some data is better than no data, but swing back to trying to get certified results later
* Something else

@dwillis, do you have any thoughts on this?

**Comment by dwillis** [comment]
@ghing I think checking with clerks to see if Clarity represents official results is a good plan. If they are not official, then when requesting precinct-level results from counties let's ask for as far back as they have them.

**Comment by ghing** [comment]
I called the Ballard County Clerk and she said that in her county, the official results are entered into a book kept in a vault at the county records office.  The results are transferred to the state electronically (presumably the Clarity) system on election night.  The clerk said the results in the book should match the ones on election night.

But ...

I called the Secretary of State's Office of Elections for clarification about what might not be in the Clarity system.  They gave examples such as provisional ballots, or a precinct that is missed when entering data on election night.

Apparently they clear things like this up 2 to 2 1/2 weeks after the election when they certify the results.

I asked if updates to the results are stored in any kind of database, and the SoS representative said no.  However, she said the state board of elections gets a hard copy of vote totals at the precinct level.

I called the State Board of Elections but just got someones voicemail.  I left a message and will update this ticket when they call back.  If I don't hear from them, I'll try calling again later this week.

Big takeaways:

* The results in Clarity really aren't official and likely don't include things like provisional ballots
* The state may have results, in some form (though that form is likely to be paper), that are official at the precinct level.  Probably easier to file records request for all counties from state board of elections rather than separately for each county.

**Comment by dwillis** [comment]
Thanks for this, @ghing. I agree with your assessments.

**Comment by ghing** [comment]
I was able to reach Maryellen Allen, Executive Director of the Kentucky State Board of Elections and she was super-helpful.  She said that there aren't really official precinct-level results.  Most counties "auto-upload" precinct-level results to the election night reporting (ENR) system which I believe is Clarity after 2010 and a state-run system pre-2010.  Some counties that can't auto-upload or ran into a snag and couldn't for a particular election won't have these results. 

The state/county certifies county-level totals for each county.  

It would be up to every county as to how they handle differences between any auto-uploaded results.  For example, provisional votes may be tallied and added to the total that is certified but are likely not allocated to precincts.

Counties must also submit paper recapitulation sheets.  These are officially submitted but not certified.  They have precinct-level results which, in many cases will just be a printout from the same system that auto-uploads results.   

If I wanted to get all the recap sheets, I would have to file a records request to the Records Officer:

Sandy Milburn
140 Walnut St.
Frankfort, KY

As for fees KY can charge up to $0.10 per page, but not for employee time responding to the records request.  Allen wasn't sure how this would play out for recapitulation sheets that were stored electronically.  It seems depending on election year and county, some recap sheets are stored as paper records while others have been scanned.

Allen suggested comparing the totals of the precinct-level results with the certified county totals to get a sense for how much they differed.  

I see a few paths forward:

* Forget precincts.  They're not official so we don't care.
* Grab the ENR precinct results for now.  For bonus points compare them with county results to determine level of exposure.  Perhaps get recap sheets or results from the counties for elections that have large discrepancies between ENR totals (aggregated from precinct-level results) and certified results for the county
* Request recap sheets for everything
* Request recap sheets for just counties/elections that don't have machine-readable ENR results.
* Proceed with requesting precinct-level results from the individual counties, even if they're not "official" or at least reaching out to each county to figure out if their precinct-level results are more complete than what they uploaded to the ENR system

**Comment by dwillis** [comment]
Thanks, @ghing. Of those paths, I think the combination of grabbing the ENR precinct results and requesting recap sheets for those counties that don't have ENR stuff seems like the best way forward.

**Comment by ghing** [comment]
@dwillis, are folks still using that Django app to track elections and availability of results?  If not, I can track that elsewhere, but either way, I want to track it.

**Comment by dwillis** [comment]
@ghing Yes, although we have it setup so that we track election information at a statewide level, not at the county-level. But the admin is definitely still there. Do you need me to send you your credentials for it?

**Comment by ghing** [comment]
I still have credentials, I think.  I'll revisit the system and see if it makes sense to try to use it or just track it in a doc somewhere.

**Comment by dwillis** [comment]
Sounds good. We also have used Github Issues to do similar things.

**Comment by dwillis** [comment]
@ghing Another project has obtained precinct-level data from the state for most counties. A link to the zip file containing them is here: https://www.strongspace.com/shared/fd78ayu8al
